.github/workflows/crowdin-ci.yml

```diff
@@ -0,0 +1,60 @@
+name: Crowdin CI
+
+on:
+  schedule:
+    - cron: "20 4 1 * *" # Runs at 4:20 AM on the first day of every month
+  workflow_dispatch: # Can be dispatched manually
+
+jobs:
+  create_approved_language_bucket_prs:
+    runs-on: ubuntu-latest
+    env:
+      CROWDIN_API_KEY: ${{ secrets.CROWDIN_API_KEY }}
+      CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID }}
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+    steps:
+      # Set up environment
+      - name: Check out code
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Install ts-node
+        run: yarn global add ts-node
+
+      - name: Set up git
+        run: |
+          git config --global user.email "actions@github.com"
+          git config --global user.name "GitHub Action"
+
+      - name: Fetch latest dev
+        run: git fetch origin dev
+
+      - name: Authenticate GitHub CLI
+        run: |
+          echo ${{ secrets.GITHUB_TOKEN }} | gh auth login --with-token
+
+      # Build and download translations to ./crowdin
+      - name: Build Crowdin project
+        uses: crowdin/github-actions@v1
+        with:
+          download_translations: true
```

Comment:
According to copilot, the `crowdin/github-actions@v1` should download the lang folders to `./crowdin`, ie. `./crowdin/fa/**`

Reply:
Sadly after testing, this action does not do what we need it to do. It requires uploading files, and then will only download the translations for those files and place them in a single PR. Placing back in draft until reversion of `crowdin/github-action@v1` usage.